packages/shell-api/src/mongo.ts

```diff
@@ -79,6 +80,11 @@ export default class Mongo extends ShellApiClass {
     }
     this._readPreferenceWasExplicitlyRequested = /\breadPreference=/.test(this._uri);
     if (fleOptions) {
+      if (fleOptions.bypassAutoEncryptionFully !== undefined) {
```

Comment:
I think we’ll want to turn this concept into an officially supported option, otherwise there would currently be no way to get a `KeyVault` or `ClientEncryption` object without (enterprise-only) automatic encryption enabled

Reply:
I'd also be fine with doing that - the key thing we don't want to expose is auto encryption but IMHO there's nothing against using the encryption primitives (KeyVault / ClientEncryption) directly... Probably clarify with @mmarcon ?

Reply:
Can I get a 5 minutes overview of what is happening here and what y'all are suggesting we do?

Reply:
So the suggestion is to enable access to instances of `ClientEncryption` and `KeyVault` event if the mongo instance was not created with auto-encryption enabled. This will allow us to better test things but of course does not make auto-encryption available to everyone (it's still the enterprise only thing).

Reply:
Sounds good to me, I don't see any harm in doing so.

Reply:
Ok, I’ll try to come up with a good name + open a separate ticket/PR so that this can be documented as public API